README.md

```diff
@@ -19,6 +19,35 @@ Alternatively, build it with `cargo build --release`, which places the binary in
 
 Run the program with the `--help` flag to see all supported arguments.
 
+```shell
+$ cargo run --release -- --help
+Extract article HTML from Wikipedia Enterprise HTML dumps.
+
+Expects an uncompressed dump connected to stdin.
+
+Usage: om-wikiparser [OPTIONS] <OUTPUT_DIR>
+
+Arguments:
+  <OUTPUT_DIR>
+          Directory to write the extracted articles to
+
+Options:
```

Comment:
Options imply that they are... optional. Will it work without options?

Reply:
Yes, but it won't extract any articles. Providing one or both of `--wikidata-ids` and `--wikipedia-urls` will extract matching articles.